javascript/1456-maximum-number-of-vowels-in-a-substring-of-given-length.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @param {string} s
+ * @param {number} k
+ * @return {number}
+ */
+
+// Time complexity: O(N)
+// Space complexity: O(1)
+
+var maxVowels = function (s, k) {
+    const vowels = new Set(['a', 'e', 'i', 'o', 'u']);
+    let left = 0;
+    let count = 0;
+    let result = 0;
+
+    for (let right = 0; right < s.length; right++) {
```

Comment:
Suggestion: Use while loop for sliding window

```suggestion
let [ left, right ] = [ 0, 0 ];

while (right < s.length) {
    if (canSlide) {
        left += 1;
    }

    right += 1;
}
```

Reply:
@aakhtar3  I have updated the code